forge/config/index.js

```diff
@@ -35,6 +36,21 @@ module.exports = fp(async function(app, _opts, next) {
         }
     }
 
+    if (process.env.npm_package_version) {
+        //npm start
+        app.log.info(`FlowForge v${process.env.npm_package_version}`)
+    } else {
+        //everything else
+        let {version} = require(path.join(module.parent.path, "..", "package.json"))
+        app.log.info(`FlowForge v${version}`)
+    }
+
+    if (!semver.satisfies(process.version,">=16.0.0")) {
+        app.log.warn(`FlowForge requires at least NodeJS v16, ${process.version} found`)
```

Comment:
I'd be tempted to move this logic right to start of the main app.js, before we try doing any setup - and console.error and exit if node isn't suitable.

Reply:
OK, makes sense. I was looking for the first place I could get hold of a `app.log` object. We can still log it here so it's clean for a good version